package.json

```diff
@@ -9,41 +9,61 @@
   "engines": {
     "node": "6.2.0"
   },
+  "config": {
+    "preset": "neutrino-preset.js"
+  },
   "dependencies": {
+    "angular": "1.5.8",
+    "angular-cookies": "1.5.8",
+    "angular-local-storage": "0.5.0",
+    "angular-resource": "1.5.8",
+    "angular-route": "1.5.8",
+    "angular-sanitize": "1.5.8",
+    "angular-toarrayfilter": "1.0.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "angular-ui-router": "0.4.2",
     "babel-preset-es2015": "6.18.0",
-    "grunt": "1.0.1",
-    "grunt-angular-templates": "1.1.0",
-    "grunt-babel": "6.0.0",
-    "grunt-cache-bust": "1.4.1",
-    "grunt-cli": "1.2.0",
-    "grunt-contrib-clean": "1.0.0",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "1.0.2",
-    "grunt-contrib-uglify": "2.0.0",
-    "grunt-usemin": "3.1.1",
-    "load-grunt-tasks": "3.5.2"
+    "bootstrap": "3.3.5",
+    "font-awesome": "4.4.0",
+    "hawk": "2.3.1",
+    "jquery": "2.1.3",
+    "jquery.scrollto": "2.1.0",
+    "js-yaml": "3.7.0",
+    "mousetrap": "1.4.6",
+    "ngreact": "0.3.0",
+    "react": "15.3.1",
+    "react-dom": "15.3.1",
+    "taskcluster-client": "1.6.3"
   },
   "devDependencies": {
-    "browserify": "^13.1.1",
+    "angular-mocks": "1.5.8",
+    "babel-preset-react": "6.22.0",
+    "clean-webpack-plugin": "0.1.13",
+    "copy-webpack-plugin": "3.0.1",
+    "enzyme": "^2.7.0",
+    "enzyme-matchers": "^2.1.0",
     "eslint": "3.8.0",
-    "grunt-eslint": "19.0.0",
-    "grunt-html-angular-validate": "0.5.8",
+    "html-webpack-plugin": "2.28.0",
+    "istanbul-instrumenter-loader": "2.0.0",
     "jasmine-core": "2.5.2",
     "karma": "1.3.0",
-    "karma-browserify": "^5.1.0",
     "karma-coverage": "1.1.1",
+    "karma-coverage-istanbul-reporter": "0.2.0",
     "karma-firefox-launcher": "1.0.0",
     "karma-jasmine": "1.0.2",
     "karma-junit-reporter": "1.0.0",
-    "karma-ng-scenario": "0.1.0",
-    "minimist": "1.2.0",
-    "ngreact-test-utils": "^1.0.3",
-    "react": "^15.3.1",
-    "react-dom": "^15.4.1",
-    "watchify": "^3.8.0"
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-webpack": "1.8.0",
+    "neutrino": "3.0.3",
+    "neutrino-preset-web": "2.1.0",
+    "react-addons-test-utils": "15.3.1"
   },
   "scripts": {
-    "test": "karma start tests/ui/config/karma.conf.js --single-run --browsers Firefox"
+    "start:stage": "NODE_ENV=development SERVICE_DOMAIN=staging neutrino start",
```

Comment:
When using Neutrino commands `start`, `build`, and `test`, the `NODE_ENV` is overridden.

- `start` sets `process.env.NODE_ENV = 'development'`
- `build` sets `process.env.NODE_ENV = 'production'`
- `start` sets `process.env.NODE_ENV = 'test'`

These aren't currently overridable.